devicon.json

```diff
@@ -75,6 +75,28 @@
         "color": "#1F0740",
         "aliases": []
     },
+    {
+        "name": "akka",
+        "altnames": ["akka-framework"],
+        "tags": [
+            "framework",
+            "java",
+            "scala",
+            "open-source"
+        ],
+        "versions": {
+            "svg": [
+                "original-wordmark",
+                "original"
+            ],
+            "font": [
+                "original",
+                "original-wordmark"
+            ]
```

Comment:
The `original` and `original-wordmark` icons aren't mono-colored, so they can't be used for our fonts.
In order to fix this you need to add separate mono-colored `plain` and `plain-wordmark` icons.

Please check out our wiki posts on [SVG-Versions](https://github.com/devicons/devicon/wiki/SVG-Versions) and [SVG-Standards](https://github.com/devicons/devicon/wiki/SVG-Standards)

Reply:
Added `plain` and `plain-wordmark`. Changed `devicon.json` accordingly.